velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -448,6 +449,24 @@ TEST_F(StringTest, overlayVarbinary) {
   EXPECT_EQ(overlay("Spark SQL", "##", -10, 4), "##rk SQL");
 }
 
+TEST_F(StringTest, repeat) {
+  const auto stringRepeat = [&](const std::optional<std::string>& str,
+                                const std::optional<int32_t>& times) {
+    return evaluateOnce<std::string>("repeat(c0, c1)", str, times);
```

Comment:
should also test string_repeat function name

Reply:
Added, thanks